src/views/Calculator.vue

```diff
@@ -0,0 +1,487 @@
+            </thead>
+            <tbody
+              v-for="(downpayments, index) in computedGetCalc"
+              :key="index"
+            >
+              <tr
+                class="
+                  my-2
+                  px-3
+                  py-2
+                  flex
+                  justify-evenly
+                  items-center
+                  space-x-20
+                "
+                v-if="downpayments.bizId == b_type.id"
+                :class="
+                  downpayments.percent == 20
+                    ? 'bg-blue-100'
+                    : downpayments.percent == 40
+                    ? 'bg-red-100'
+                    : downpayments.percent == 60
+                    ? 'bg-green-100'
+                    : 'bg-yellow-100'
+                "
+              >
+                <td class="font-bold text-lg -mr-6 -ml-10">
+                  {{ downpayments.percent }}%
+                </td>
+                <td class="font-medium text-base">
+                  {{
+                    $formatCurrency(
+                      Math.ceil(downpayments.actualDownpayment)
+                    ) || ""
+                  }}
+                </td>
+                <td class="font-medium text-base">
+                  {{
+                    $formatCurrency(
+                      Math.ceil(downpayments.actualRepayment / 12)
+                    ) || ""
+                  }}
+                </td>
+                <td class="font-medium text-base">
+                  {{
+                    $formatCurrency(Math.ceil(downpayments.actualRepayment)) ||
+                    ""
+                  }}
+                </td>
+                <td class="font-medium text-base">
+                  {{ $formatCurrency(Math.ceil(downpayments.total)) || "" }}
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import registerBg2 from "@/assets/img/register_bg_2.png";
+import AutoComplete from "@/components/Autocomplete/AutocompleteSearch.vue";
+import calculate from "../utilities/calculator";
+import { get, post } from "../utilities/api";
+import ArrowUp from "../components/svgs/arrowup.vue"
+export default {
+  data() {
+    return {
+      registerBg2,
+      apiUrls: {
+        getProduct: `/api/inventory`,
+        getCalculation: `/api/price_calculator`,
+        businessTypes: `/api/business_type`,
+        downPaymentRates: `/api/down_payment_rate`,
+      },
+      selectedProduct: {},
+      downPaymentRates: [],
+      products: [],
+      businessTypes: [],
+      product: "",
+      isActive: false,
+      repaymentCycle: 14,
+      repaymentDuration: 180,
+      calculation: [],
+      fPayment: "",
+      tpayment: "",
+      pPrice: "",
+      rPayment: "",
+      downPaymentRates: [],
+      eachDownpayment: {},
+      totals: [],
+      test1: true,
```

Comment:
Are we using this?

Reply:
I wil go through all the data property and remove the ones we are not using